prod/tekton-pipelines/config/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "app": {
-        "name": "ks-cli-pipelines-apps",
+        "name": "ks-pipelines-apps",
```

Comment:
`ks-cli` has a more clear meaning. It represents the KubeSphere CLI repo instead of the KubeSphere core. So, I prefer to do not to change it.

Reply:
But this folder would contain many other repositories in `kubesphere-sigs` group which are not related `cli`. Before this, there only `kubesphere-sigs/ks` was added here, so calling the name as `ks-cli-pipeline-apps` was okay to me.

Reply:
Got it.